### DISCUSSION TITLE
grid-material右键移动时物体与地面网格错位严重

### DISCUSSION THREAD

**Discussion by geekroscom** [galacean/discussion#17] [Engine Toolkit]
当前引擎版本：0.8.0-beta.30
当前toolkit版本：0.8.0-beta.8

问题描述及复现：

场景中包含一个无限网格地面和一个立方体实体，使用鼠标右键拖动场景时，立方体与无限网格地面和地面错位严重，当窗口处于正常最大化时候，错位不明显，**可缩小窗口（或固定小窗口）测试复现**。

该程序可复现问题：https://gitee.com/geekros/Geek_World_Test

程序说明：

1、对grid-material进行了简单的移植，移植后的程序位于：https://gitee.com/geekros/Geek_World_Test/tree/master/src/world/package/ground_plane
2、在程序的第101行调用了封装后的grid-material，程序详见：https://gitee.com/geekros/Geek_World_Test/blob/master/src/world/index.ts

**Comment by eyworldwide** [comment]
@yangfengzzz 关注一下

**Reply by yangfengzzz** [reply]
已经给他单发了录屏，维勒测试下来也没有问题
